plugins/custom-cols.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/webofmars/kubectl-custom-cols/archive/v0.0.2.tar.gz
+    sha256: "f01d1aa63c320989071baa88d7b0a8fefa1147e0da8b9edf32874def21c84736"
+    files:
+    - from: "./kubectl-custom-cols-0.0.2/kubectl-custom-cols"
```

It would keep the noise down for future manifest revisions, if you could change your archive to use a static path (without the version string), like so:
```suggestion
    - from: "./kubectl-custom-cols"
```

---

Humm. i added these from my tests using GH generated release and installing with krew (only overriding the manifest path). It seems that GH adds this folder when building a tar.gz release. Previous was what you suggested. 

Unless i missed something ? 🤔

---

fixed with a `kubectl-custom-cols-*` pattern